EIPS/eip-7231.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: 7231
+title: Identity aggregated NFT
+description: The aggregation of web2 & web3 identities to NFTs, authorized by individuals, gives attributes of ownerships, relationships, experiences.
+author: Chloe Gu <chloe@carv.io>, Navid X. (@xuxinlai2002), Victor Yu <victor@carv.io>, Archer H.
+discussions-to: https://ethereum-magicians.org/t/eip-7231-namespaced-storage-layout/14796
```

Comment:
Flagging this, the discussions-to link points to discussion on eip-7201 and not eip-7231.

@xuxinlai2002 might want to check this once.

Reply:
Thanks ,it has been revised.
https://ethereum-magicians.org/t/erc7231-identity-aggregated-nft/15062